skbio/stats/gradient.py

```diff
@@ -428,7 +430,7 @@ def __init__(self, coords, prop_expl, metadata_map,
                              % (len(prop_expl), axes))
 
         # Restrict coordinates to those axes that we actually need to compute
-        self._coords = coords.ix[:, :axes-1]
+        self._coords = coords.loc[:, :axes-1]
```

Comment:
This should be `iloc`

Reply:
It doesn't work with `.iloc` because the column names are integers.

Reply:
A little more detail: for whatever reason, using `.loc` here with integer column names matches the behavior of `.ix` so I decided to go with that.

Reply:
![](https://media.giphy.com/media/9Y5BbDSkSTiY8/giphy.gif)